addons/l10n_be_pos_restaurant/models/pos_config.py

```diff
@@ -0,0 +1,49 @@
+from odoo import api, models
+
+class PosConfig(models.Model):
+    _inherit = 'pos.config'
+
+    def _is_belgian_fp_company(self):
+        return self.env['account.fiscal.position'].search_count([('company_id', '=', self.env.company.id), ('country_id.code', '=', 'BE')]) > 0
+
+    def _create_takeaway_fiscal_position(self, config):
+        ChartTemplate = self.env['account.chart.template'].with_company(self.env.company)
+        tax_21 = ChartTemplate.ref('attn_VAT-OUT-21-L', raise_if_not_found=False)
+        tax_12 = ChartTemplate.ref('attn_VAT-OUT-12-L', raise_if_not_found=False)
+        tax_6 = ChartTemplate.ref('attn_VAT-OUT-06-L', raise_if_not_found=False)
+
+        if tax_21 and tax_12 and tax_6:
+            fp = self.env['account.fiscal.position'].create({
+                'name': 'Take out',
+            })
+            self.env['account.fiscal.position.tax'].create({
+                'tax_src_id': tax_21.id,
+                'tax_dest_id': tax_6.id,
+                'position_id': fp.id
+            })
+            self.env['account.fiscal.position.tax'].create({
+                'tax_src_id': tax_12.id,
+                'tax_dest_id': tax_6.id,
+                'position_id': fp.id
+            })
+            config.write({'takeaway': True, 'takeaway_fp_id': fp.id})
+
+    @api.model
+    def load_onboarding_bar_scenario(self):
+        super().load_onboarding_bar_scenario()
+        if self._is_belgian_fp_company():
+            ChartTemplate = self.env['account.chart.template'].with_company(self.env.company)
+            tax_alcohol = ChartTemplate.ref('tax_alcohol_luxury')
```

Comment:
`tax_alcohol_luxury` will only exist if `(company.chart_template or '').startswith('be')`. I'm not sure why you are checking the fiscal positions with `_is_belgian_fp_company` :thinking: